sessionctx/variable/tidb_vars.go

```diff
@@ -741,6 +741,10 @@ const (
 	// TiDBMaxAutoAnalyzeTime is the max time that auto analyze can run. If auto analyze runs longer than the value, it
 	// will be killed. 0 indicates that there is no time limit.
 	TiDBMaxAutoAnalyzeTime = "tidb_max_auto_analyze_time"
+	// TiDBFastDDL indicates whether use lighting to help acceleate adding index stmt.
+	TiDBFastDDL = "tidb_fast_ddl"
+	// TiDBDiskQuota used to set disk quota for lightning add index.
+	TiDBDiskQuota = "tidb_disk_quota"
```

Comment:
How about using `executor.GlobalDiskUsageTracker` to track the memory usage?

Reply:
executor.GlobalDiskUsageTracker is for executor query check storage usage?
TiDBDiskQuota has two functions: 1) used to set a low bar to make sure there is at least disk space to enable lightning execution enviroment.  2) it also could let user change the lightning local disk quota to make add index lightning do partial ingest sst to TiKV. 
I think it has risk to mix merge TiDBDiskQuota into executor.GlobalDiskUsageTracker or could we delay this task into next stage if needed
Also I found the name is not very clear to show it meaning, then I will change it to TiDBDDLDiskQuota, is that ok?